net/tstun/wrap_test.go

```diff
@@ -310,8 +312,9 @@ func TestFilter(t *testing.T) {
 		drop bool
 		data []byte
 	}{
-		{"junk_in", in, true, []byte("\x45not a valid IPv4 packet")},
-		{"junk_out", out, true, []byte("\x45not a valid IPv4 packet")},
+		{"short_in", in, true, []byte("\x45xxx")},
+		{"short_out", out, true, []byte("\x45xxx")},
+		{"ip97_out", out, false, must.Get(hex.DecodeString(strings.ReplaceAll("4500 0019 d186 4000 4061 751d 644a 4603 6449 e549 6865 6c6c 6f", " ", "")))},
```

Comment:
I note that packet_test.go defined a noWS() function to do this for all unicode whitespace runes, though I don't see why it was needed there.

Reply:
made them both consistent with a new mustDecodeHex func. maybe I'll move that to some packet test helper package later.